packages/eslint-config-small-ads/.prettierrc

```diff
@@ -0,0 +1,11 @@
+{
+  "bracketSpacing": true,
+  "jsxBracketSameLine": false,
+  "jsxSingleQuote": false,
+  "printWidth": 80,
+  "proseWrap": "always",
+  "semi": false,
+  "singleQuote": false,
+  "tabWidth": 2,
+  "trailingComma": "all"
+}
```

Comment:
After we install prettier as devDep on the root, can we try to delete this? I believe this isnt needed for now.

Reply:
I thought we would need this to tell prettier how to behave. I mean, I thought eslint would use whatever prettier would advice whenever they would have clashing rules.

Reply:
But I installed in the root, and removed the file like you suggested. I suppose everything is being handle in the `index.js` from `eslint-config-small-ads` but it is still a bit confusing.

Reply:
Lets get this right the next days. We can have a talk, but first lets figure out why we have different quotes.